src/GraphRequestUtil.ts

```diff
@@ -17,7 +17,15 @@ export const oDataQueryNames = ["$select", "$expand", "$orderby", "$filter", "$t
 /**
  * To hold list of the service root endpoints for Microsoft Graph and Graph Explorer for each national cloud.
  */
-export const graphURLs = ["https://graph.microsoft.com", "https://graph.microsoft.us", "https://dod-graph.microsoft.us", "https://graph.microsoft.de", "https://microsoftgraph.chinacloudapi.cn"];
+export const graphURLs = new Set<string>();
+
+// using an IIFE to populate the set object with the graph host names as Set(iterable:Object) is not supported in Internet Explorer
+(() => {
+	const urls = ["graph.microsoft.com", "graph.microsoft.us", "dod-graph.microsoft.us", "graph.microsoft.de", "microsoftgraph.chinacloudapi.cn"];
```

Comment:
you could initialize the set directly from the array, see the edit in the response.
https://stackoverflow.com/a/30902786/3808675

Reply:
I used an IIFE with set.add() instead as IE does not support Set(array). IE creates an empty set when I passed in new Set(object/array).

Reply:
interesting, and I'm guessing this is an edge case the transpilation is not catching?

Reply:
additionally this is something that people (or we) can polyfill using core-js, see the first example. https://www.npmjs.com/package/core-js